benchmarks/python/test_matmul.py

```diff
@@ -25,21 +25,32 @@ def load_matmul_problems():
         return list((int(m), int(n), int(k), layout) for m, n, k, layout in reader)
 
 
+@pytest.mark.parametrize("half_reduction", [False, True], ids=["fullred", "halfred"])
+@pytest.mark.parametrize("eager", [False, True], ids=["nvfuser", "eager"])
+@pytest.mark.parametrize("dtype", [torch.float16, torch.bfloat16], ids=["fp16", "bf16"])
 @pytest.mark.parametrize(
-    "config", load_matmul_problems(), ids=lambda val: "_".join(str(v) for v in val)
+    "config", load_matmul_problems(), ids=lambda val: "-".join(str(v) for v in val)
 )
-@pytest.mark.parametrize("dtype", [torch.float16, torch.bfloat16])
 def test_matmul_nvf_benchmark(
     benchmark,
+    eager: bool,
```

Comment:
If we separate `eager` benchmark into its own benchmark, they will not run by default (https://github.com/NVIDIA/Fuser/blob/5a6c90b17c950a922697ca8f7955f8c1c3cb55ed/benchmarks/python/conftest.py#L107-L116). Right now, we will run both nvfuser (which is ATen at this time) and eager in every run.

We can separate out the common code into a utility function and call them from `test_matmul_nvf_benchmark` and `test_matmul_baseline_benchmark`.

Reply:
When I split the test into two tests there will be no difference between eager and compile, but my understanding is that we need to have both of those to trigger this code path.

Reply:
I guess I can just put that `compile` option in, since eventually we will probably extend this to cover some epilogue cases, and multi-matmul cases

Reply:
The second function will have the compile parameter with just the `[False]` value to exercise the eager benchmark and it will not be run by default.

>there will be no difference between eager and compile, but my understanding is that we need to have both of those to trigger this code path.

I am not sure what you mean here -- trigger which code path?

Reply:
The `compile` parameter is what is required to skip eager by default.

Reply:
Ah right. Gotcha. I will make that change tonight/AM eastern

Reply:
I just pushed the change splitting the baseline into a separate test that can be enabled with `--benchmark-eager`.